scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb

```diff
@@ -4,15 +4,63 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Mart Tutor Assistant\n",
+    "# Math Tutor Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
+    "\n",
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n",
```

Comment:
Needs objective + link to documentation

Reply:
# Math Tutor Assistant

**Objective**

This notebook demonstrates the following:

- Showcases the foundational concepts of Assistants such as Threads, Messages, Runs, Tools, and lifecycle management.

This tutorial uses the following Azure AI services:

- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)\n",
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:

- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)
- [Github-OpenAI Python/examples/Assistant demo notebook](https://github.com/openai/openai-python/blob/main/examples/assistant.py)

## Time

You should expect to spend 5-10 minutes running this sample. 

## About this example

The objective of the provided Python file is to create an Azure OpenAI Assistant named "Math Tutor" using the Azure OpenAI API. The assistant is designed to function as a personal math tutor, capable of answering math questions through code interpretation. The script initiates a conversation with the assistant, guiding it through various mathematical queries and scenarios to showcase its capabilities.

This sample provides developers with a clear demonstration of how to integrate the Assistants API into their projects, highlighting its simplicity and effectiveness in leveraging foundational concepts.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/6833b0fb2ec9b571ea2f9fd191cde16e56bf4bbc